app/validation/validator.py

```diff
@@ -282,6 +283,8 @@ def get_defined_numeric_value(defined_value, system_default, answer_ranges):
             referred_answer = answer_ranges.get(defined_value['answer_id'])
             if referred_answer is None:
                 values = None  # Referred answer is not  valid (picked up by _validate_referred_numeric_answer)
+            elif referred_answer.get('default') is not None:
```

Comment:
@jonnyshaw89 can this just be elif referred_answer.get('default')

Reply:
No because otherwise `0` would evaluate